modules/ingress/README.md

```diff
@@ -1,61 +1,39 @@
 # Module setup ingress controller.
 
-# Example 1. Minimal parameter set and create ingress on default annotations
+## Problem
+When creating R53 records or CDN origins we need to explicitly pass ALB's DNS name to those resources. It's not convenient and not flexible.
```

Comment:
Please ask someone to give feedback on the text, I do know the concept and can understand but not sure if everyone will understand.